### ISSUE TITLE
Scanners with spaces in the name lead to "ERROR : Option format invalid."

### ISSUE THREAD

**Issue by linuxundich** [janrueth/es2button#4]
In example-epsonscan2.sh you call epsonscan2 with e.g.

    epsonscan2 -s ${ES2_DEV} $(dirname $0)/a4_preset.SF2

this fails on my system with a Epson ES-60W since epsonscan2 lists my Epson ES-60W with a space in the name.

```
$ epsonscan2 -l
 === List of available devices ==
device ID :Epson ES-60W:003:016
ModelID:ES0181
```

I have to add quotes to the call:

    epsonscan2 -s "${ES2_DEV}" $(dirname $0)/a4_preset.SF2

Without the quotes, I receive the error "Option format invalid":

```
$  /usr/bin/es2button /usr/share/es2button/example/merge-pdfs.sh
[...]
+ epsonscan2 -s Epson ES-60W:003:017 /usr/share/es2button/example/test.SF2
ERROR : Option format invalid.
+ echo 'Finished scan'
Finished scan
Resuming from script


```

**Comment by janrueth** [comment]
yeah, the space is the problem and your fix is correct. Will put up a fix soon.

**Comment by janrueth** [comment]
fixed in v0.2.1
